### ISSUE TITLE
Mission initializes, but doesn't seem to progress. Retaliations are non existant, and we only ever fight troops.

### ISSUE THREAD

**Issue by Novau101** [LordGolias/antistasi#79]
With the plethora of content that antistasi original and RHS has to offer, I have created a multiplayer dedicated server to facilitate this game mode - but we've been playing for 8 hours ish straight with nothing but basic troops and very little challenge to counteract. Objective taking is easy. Was wondering if we could solve this?

**Comment by mike-187** [comment]
your mission started on a dedicated server? what mission are you running Tanoa or Altis? I tried Tanoa Yesterday and it did not init. Will try Altis just to be sure...

**Comment by LordGolias** [comment]
I think @Novau101 is running the master version. @mike-187, that is working, right?

**Comment by mike-187** [comment]
Yes the master version is working. I also tested init/.... and other branches before the merge and they also were not working. (Had a copy on disk)

**Comment by Novau101** [comment]
Ho. Sorry for the late reply. I've since moved onto the community antistasi instead.

We made a dedicated server on Altis and simply many core elements weren't functioning very well. It could be down to the pbo not functioning correctly but it has done for other game modes so I don't know. Taking objectices would do nothing except backend stats changing, it wouldn't spur reinforcements - there would be no retaliation and it was pretty boring - it almost was like someone put units on the map, but just had them guard that area. There didn't seem to be an ai commander as such.

**Comment by LordGolias** [comment]
the reason you mention is that AAF takes time to build its arsenal. This means that in the beginning, things are mostly infantry. We could speed up the process, or provide a difficulty level that makes AAF more agressive. We have not focused on this because there are still basic things being worked out, but if you have specific suggestions, feel free to propose them.

**Comment by LordGolias** [comment]
That is an error. When I played (MP non+dedicated) we were immediately attacked after we captured a powerplant. Was that in dedicated MP? Do you have the logs?

Thanks!

**Comment by Novau101** [comment]
I unfortunately do not, and I have since removed the mod from our server as we were finding it frustrating. A problem more on our end I'm sure! We liked many of the improvements you had made, but simply wanted a stable experience and found that in the community antistasi.

A couple of things we noticed also, not sure if intentional or part of the bug.

Missions did not appear randomly.
Reinforcements did not happen.
Missions did not continue - for example secure supply truck. Securing did not further quest to second part.
Could not scroll on right pane of mission window.
Mission window cut off longer mission names.
Saves didn't work after certain time regardless of garbage command. Spent 30 mins saving plus.
Absolutely NO vehicles bar troop trucks.
Checkpoints contained no static turrets.
Taking locations changed colour of map icon, but did not create a pop up saying "captured xxx" etc.

There may be more - but I thought it worth mentioning in the future if this issue pops up again for ease of identification

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by LordGolias** [comment]
> We liked many of the improvements you had made, but simply wanted a stable experience and found that in the community antistasi.

Yap, I definitely agree with you that atm, this mod is way less stable than the original antistasi. We are not quite there yet, as essentially most of the code had to be re-written in order to accommodate what we changed. This naturally brings instability because it was not as tested as the other code.

Thank you for enumerating the errors. They are certainly things that we will be focusing once the initialization with different factions is stable.

**Comment by Novau101** [comment]
I would love to help out when you deem it more stable, and would love to help in development ideas or development itself. There are many things the core antistasi does not do that I think Is a shame as it's potential is tremendous.

A few example are

Periodic patrolling jets

Reinforcements by randomised means. Possible options include apc, helo, paradrop, spec ops team, mortar/artillery emplacements within x radius.

More defined convoys/road patrols to ambush between towns.

More water based missions

The ability for you and the enemy to utilise every vehicle and weapon in the game (effectively) with appropriate cost and balance.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
